magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -223,9 +229,21 @@ def run_with_flags(generator):
   # files.
   date_and_time = time.strftime('%Y-%m-%d_%H%M%S')
   digits = len(str(FLAGS.num_outputs))
+
   for i in range(FLAGS.num_outputs):
     generated_sequence = generator.generate(input_sequence, generator_options)
 
+    if FLAGS.exclude_primer_midi:
+      # Remove any notes with start_time less than start_time of generated
+      # sequence. Otherwise, subtract length of removed input_sequence from
+      # start_time and end_time of remaining notes.
+      for note in generated_sequence.notes[:]:
+        if note.start_time < start_time:
+          generated_sequence.notes.pop(0)
```

Comment:
Yeah, I think `remove` should be fine too. `del` might be slightly more performant because it won't have to search for the object to remove, but in this context, it shouldn't matter.